manage.py

```diff
@@ -0,0 +1,10 @@
+"""
+ona-oicd manage.py file, used for testing
+"""
+import os
+
+from django.core import management
+
+os.environ["DJANGO_SETTINGS_MODULE"] = "tests.settings"
```

Comment:
Do we wanna have `tests.settings` as our default settings?

Reply:
This is only used for tests, this won't run or be an issue when this application is imported into another.

Reply:
Our default settings are in `oicd.settings`.

Reply:
_Excluded the entire "tests" folder from the setup in the latest commit_